lib/loadMochaOptsIntoArgs.js

```diff
@@ -0,0 +1,18 @@
+// TODO: mocha should be a peer dependency, and we should make sure that we
+//       load mocha from the project root (resolve-cwd module?). The file
+//       `mocha/bin/options` was introduced in mocha version 2.2.4 - so our
+//       peer dependency should reflect that.
+const getOptions = require("mocha/bin/options");
+
+module.exports = function loadMochaOptsIntoArgs() {
+  // getOptions is used by mocha to expand process.argv with any information
+  // that is found in a related mocha.opts file. The mocha.opts file may be
+  // referenced in the arguments present in progress.argv so in order to avoid
+  // extending it twice the environment variable is set.
+
+  if (!process.env.LOADED_MOCHA_OPTS) {
+    getOptions();
+  }
+
+  return process.argv;
+};
```

Comment:
@alexjeffburke this file is just calling into mocha's options loader. The function is here to make sure that we don't need to pollute the rest of the project with intimate knowledge about mocha's internals (at least not regarding this) - mocha's `getOptions` is not safe against being called more than once and relies on the caller checking the presence of the environment variable.